packages/ai/index.ts

```diff
@@ -0,0 +1,3 @@
+import * as ai from "./internal/ai";
+
+export default ai;
```

Comment:
Matches what we do for built-ins:

```suggestion
// Supports:
// import { chat, ChatOpts } from '@airplane/ai'
export * from "./internal/ai";

// Supports:
// import ai from '@airplane/ai'
export * as default from "./internal/ai";
```